cmd/canary/main.go

```diff
@@ -32,19 +32,30 @@ func main() {
 		err = fmt.Errorf("SAMPLE_INTERVAL is not a valid integer")
 	}
 
+	timeout := 0
+	defaultTimeout := os.Getenv("DEFAULT_MAX_TIMEOUT")
+	if defaultTimeout == "" {
+		timeout = 10
+	} else {
+		timeout, err = strconv.Atoi(defaultTimeout)
+		if err != nil {
+			err = fmt.Errorf("DEFAULT_MAX_TIMOEUT is not a valid integer")
```

Comment:
Actually, nevermind. 

the assignment of 'err' here is useful for when the environment variable is not set to a valid int. 

`timeout` is set below for canary.Config.MaxSampletimeout and is used as the sensor timeout value if the target interval is greater than the max timeout value.